net-proxy/3proxy/3proxy-0.8.12.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
7 should work here (please test reinstalling after bumping)

Reply:
Done. If it's recommended for new ebuilds then why skel.ebuild contains EAPI=6?

Reply:
That's pretty interesting actually. If you make a new ebuild using vim, it gives you EAPI=7 and an empty BDEPEND line. But the /skel.ebuild has EAPI-6 indeed. Maybe the one in tree has been forgotten, or it's held back because **some few** eclasses don't support 7 yet. Still it does say: 
```
# It is suggested that you use the latest EAPI approved by the Council.
```

I might open a bug about updating the /skel.ebuild, thanks!